web/src/routes/ProjectView/MapView/selector.ts

```diff
@@ -22,8 +22,19 @@ const selectRenderProps = createSelector(
   ({ state }: InputType) => state.ui.viewport.translate,
   ({ state }: InputType) => state.ui.screensize.width,
   ({ state }: InputType) => state.ui.screensize.height,
-  ({ state, activeProject }: InputType) =>
-    state.projects.projectMeta[activeProject],
+  ({ state, activeProject }: InputType) => {
+    return {
+      creatorAgentPubKey: state.agentAddress,
+      createdAt: Date.now(), // f64
+      name: 'project',
+      image: '',
+      passphrase: 'test',
+      isImported: false,
+      priorityMode: PriorityMode.Universal,
+      topPriorityOutcomes: [],
+      actionHash: '',
+    } as WithActionHash<ProjectMeta>
+  }, // state.projects.projectMeta[activeProject],
```

Comment:
this will require a proper solution that works across both Acorn runtime contexts